src/questions/flow.js

```diff
@@ -31,6 +29,7 @@ export const postInstall = async ({ answer, answers: { appname, mobile } }) => {
     }
 
     if (!mobile) {
+        // TODO breaks atm. command not found: flow
```

Comment:
Fix it!
http://www.artima.com/intv/fixit2.html

Reply:
OK maar serieus, het hoeft niet meteen maar in het vervolg is het handiger om problemen direct te fixen zodra je ze tegenkomt zodat je een hoge kwaliteit bewaart.
Op zich is het wel al heel goed dat je een comment heb neergezet though.